recipes/pylibftdi/LICENSE.txt

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2010-2020 Ben Bass
```

Comment:
This file is already included with the PyPI tarball so you shouldn't include it manually in the recipe.

Reply:
Thanks for the quick review. I removed the file and the reference to it in `meta.yaml`.

Reply:
And now I get the conda-forge-linting error "license_file entry is missing, but is required". How can I solve that?

Reply:
Put back `license_file: LICENSE.txt` (and only that one line) 😉